apps/newsletters-api/open-api.yaml

```diff
@@ -7,7 +7,7 @@ info:
     email: newsletters-dev@guardian.co.uk
 
 paths:
-  /v1/newsletters:
+  /newsletters:
```

Comment:
Is it worth flagging somehow that this represents the legacy version?  Otherwise it's not clear (to me) what the difference is between `/newsletters` and `v1/newsletters`, as both have the same summary and description

Reply:
You're right - I'm not sure what the right way to describe it is though. 🤔

Reply:
Suggestion (intended to be tweaked):
summary: Suggestion for summary: Returns a list of newsletters (legacy api)
description: Include current and previous (cancelled or paused) editorial newsletters (in legacy format). Does not include marketing or product newsletters.